test/parallel/test-http-client-abort2.js

```diff
@@ -31,6 +31,8 @@ server.listen(0, common.mustCall(() => {
   const options = { port: server.address().port };
   const req = http.get(options, common.mustCall((res) => {
     res.on('data', (data) => {
+      req.on('abort', common.mustCall());
```

Comment:
Does this actually test the change? As far as I can tell `'abort'` was emitted only once even before. Can we have a better test?

Reply:
What do you mean? I call `abort()` twice and `'abort'` is only emitted once...

Reply:
Yes, that's my point. it works like this even without the changes proposed here, so it is not clear what this is testing.

Reply:
```js
'use strict';

const http = require('http');

const server = http.createServer(function(req, res) {
  res.flushHeaders('foo');
});

server.listen(function() {
  const req = http.get({ port: server.address().port });

  req.on('response', function(res) {
    const dump = res._dump;

    res._dump = function() {
      console.log('dump');
      dump.apply(res, arguments);
    };

    req.abort();
    req.abort();
    req.abort();
    req.abort();
  });

  req.on('abort', function() {
    console.log('abort');
  });
});
```

`'abort'` is already emitted only once. The difference is that `res._dump()` is only called once now. Hope it makes sense.

Reply:
Makes sense. Updated. Not pretty but I can't see any other way.